commands.go

```diff
@@ -3946,3 +3948,25 @@ func (c cmdable) ModuleLoadex(ctx context.Context, conf *ModuleLoadexConfig) *St
 	_ = c(ctx, cmd)
 	return cmd
 }
+
+func (c cmdable) ACLLog(ctx context.Context, count ...int64) *ACLLogCmd {
```

Comment:
Why is "...int64" used here? ACL-LOG has only one parameter.  `acl log 10` or `acl log rest`.

Reply:
I think this is so we can either add or omit the count parameter, since it's not required. So we could do either of the following:

```
redis.ACLLog(ctx)           // For all log entries
redis.ACLLog(ctx, n)       // For n log entries
redis.ACLLogReset(ctx)  // For resetting log entries
```